docs/excel/custom-functions-debugging.md

```diff
@@ -166,7 +166,7 @@ There are several build tasks available.
 You can use the following tasks to start debugging on desktop or online.
 
 - `npm run start:desktop`: Starts Excel on desktop and sideloads your add-in.
-- `npm run start:web`: Starts Excel on the web and sideloads your add-in.
+- `npm run start:web -- --document {url}` (where `{url}` is the URL of an Excel file on OneDrive or SharePoint): Starts Excel on the web and sideloads your add-in.
```

Comment:
Given that you define the `url` parameter a few lines up, I'm not sure this second definition is necessary. It clutters the nice list you have going here.
```suggestion
- `npm run start:web -- --document {url}`: Starts Excel on the web and sideloads your add-in.
```

Reply:
The two places where the parameter is defined are in different sections and I think some people may be reading this section that didn't read the earlier one. So, I'd like to keep this.